src/conventional/changelog.rs

```diff
@@ -195,6 +199,8 @@ impl<'a> ContextBuilder<'a> {
             config.release_commit_message_format.as_str(),
         )?;
         handlebars.register_template_string("user_url_format", config.user_url_format.as_str())?;
+        let (host, owner, repository) = Self::host_info(git)?;
```

Comment:
this method is called for each commit, creating a new context builder should not create a new handlebars registry, nor calculate host/owner/repository, or the builder should be reusable